50.md

```diff
@@ -47,4 +47,8 @@ Relays SHOULD exclude spam from search results by default if they support some f
 
 Relay MAY support these extensions:
 - `include:spam` - turn off spam filtering, if it was enabled by default
-- `domain:<domain>` - include events from users whose valid nip05 domain matches the domain
+- `domain:<domain>` - include only events from users whose valid nip05 domain matches the domain
+- `topic:<topic>` - include only events within a classified topic (short string, one to two words)
```

Comment:
How it topic different from hashtag, which you can get with `#t` filters? It seems too niche - I would remove it.

The other ones look good to me.

Reply:
Unless somehow this is supposed to find things that in theory could be tagged with a hashtag but aren't? :thinking: How would that work?

Reply:
We are doing our own topic classification after the event has been posted based on the content of the event and the links. It has nothing to do with hashtags created by the user.

Reply:
See https://api.nostr.wine/classification?topic=food as an example.

Reply:
Good question and I don't have an obvious solution. I was thinking we'd leave it open ended because different providers will likely have different classified topics.

For nostr.wine specifically, we could expose an API with the most popular classified topics.

Reply:
Cool. I would personally omit this one from the PR until that's sorted out. I think it's too ambiguous for other implementations, whereas the other extensions you proposed are unambiguous and self-explanatory.

Reply:
Ok - done. 

I do think we need some way for clients to query for topics but I really don't want to be the one to argue about or decide where it should go.